apps/comps/components/agent-profile/index.tsx

```diff
@@ -67,6 +68,11 @@ export default function AgentProfile({
   const skills = agent?.skills || [];
   const trophies = sortTrophies((agent.trophies || []) as Trophy[]);
 
+  // Sort ranks and scores for display on agent profile & tooltip
+  const sortedRanks =
+    agent.stats.ranks?.slice().sort((a, b) => a.rank - b.rank) || [];
```

Comment:
would be better to do this sorting server-side so it can be optimized and cached.  Client-side sorting is an anti-pattern we should be moving away from wherever possible